devicon.json

```
@@ -1,4 +1,32 @@
 [
+    {
+        "name": "aerospike",
+        "altnames": [
+            "aerospike"
+        ],
```

The altname here is the same as the name. If there are no other names than the one used in the "name" attribute, you can leave altnames empty:
```suggestion
        "altnames": [],
```